app/assets/stylesheets/stores/_store_session.scss

```diff
@@ -3,10 +3,12 @@
   display: flex;
   justify-content: center;
   flex-direction: column;
+  margin: 3em 0 0 0;
 }
 
 .registration-container {
   font-family: $main-font;
+  margin: 3em 0 0 0;
```

Comment:
Unexpected longhand value '3em 0 0 0' instead of '3em 0 0'   shorthand-property-no-redundant-values